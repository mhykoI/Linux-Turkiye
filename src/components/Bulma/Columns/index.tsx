import React from 'react';

const Columns: React.FC = ({ children }) => {
  return <div className="columns">{children}</div>;
};

export default Columns;
