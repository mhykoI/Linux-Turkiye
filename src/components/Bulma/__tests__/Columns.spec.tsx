import React from 'react';
import renderer from 'react-test-renderer';
import Columns from '../Columns';

describe('<Columns />', () => {
  test('should render correctly', () => {
    const tree = renderer.create(<Columns>This is Columns!</Columns>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
