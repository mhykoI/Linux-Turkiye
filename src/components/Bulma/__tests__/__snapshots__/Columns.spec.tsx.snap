// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Columns /> should render correctly 1`] = `
<div
  className="columns"
>
  This is Columns!
</div>
`;
